src/show/pages/public/Home/index.jsx

```diff
@@ -39,7 +42,16 @@ const Home = () => {
         <div className='flex ml-5 mt-3 flex-row'>
           {constructors.length > 0 ? (
             constructors?.map(constructor => (
-              <ConstructorCard key={constructor.id} constructor={constructor} />
+              <div
+                onClick={() =>
+                  navigate(CONSTRUCTOR_PATH.replace(':id', constructor.id))
+                }
+                key={constructor.id}>
+                <ConstructorCard
+                  key={constructor.id}
```

Comment:
There's no more need for this key, only the first item needs it.

Reply:
done